students/patchcarrier/Lesson02/fizzbuzz_exercise.py

```diff
@@ -0,0 +1,14 @@
+for k in range(1,101):
+    
+    if not((k % 5) or (k % 3)):
+        print('FizzBuzz')
+    
+    elif not(k % 5):
+        print('Buzz')
+    
+    elif not(k % 3):
```

Comment:
no need for parens here, `not` is a statement not a function.

also convention is to check `k % 3 == 0`